### DISCUSSION TITLE
Documentation for the site navbar v5

### DISCUSSION THREAD

**Discussion by ValentinGratz** [twbs/bootstrap#31947] [General] [answered]
Hello,

I'm looking a bit at the documentation for what's new in version 5, and in the navbars, I still can't see the documentation for the new two-column mobile version navbar, like in the attached example.

Is this normal?

Thank you
![nav_responsive_5](https://user-images.githubusercontent.com/39189263/96698266-32e33680-138d-11eb-966b-f24fc2b4e7d6.png)

**Comment by patrickhlauke** [comment]
that's not a new component or anything. it uses standard bootstrap "bits" to achieve this.

**Comment by ValentinGratz** [comment]
Ok, without the test I just did with lots of links in the nav, it doesn't work (taking the css and js cdn from v5).

![image](https://user-images.githubusercontent.com/39189263/96700820-28766c00-1390-11eb-9c27-20f431954d1d.png)
my code

![image](https://user-images.githubusercontent.com/39189263/96700910-43e17700-1390-11eb-9411-b1a5e92772d0.png)
this resultat

**Comment by patrickhlauke** [comment, marked as answer]
not everything is pure bootstrap. you'll have to add a few extra styles yourself, building on top of bootstrap core, in your projects, just like we do for the documentation...
